build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -1015,6 +1015,8 @@ CONFIG_MPU_TEST_APP_ADDR=0x80200100
 # Binary manager
 #
 CONFIG_BINARY_MANAGER=y
+CONFIG_BM_PRIORITY_MAX=205
+CONFIG_BM_PRIORITY_MIN=200
```

Comment:
1. These values are used in mkbinheader.py and binary manager codes to validate priority so I made it to avoid mismatching between them.
2. I will add it